.github/workflows/ci-frontend.yml

```
@@ -28,6 +28,6 @@ jobs:
           pip install pytest
           pip install pytest-random-order
           pip install --no-cache-dir .
-      - name: Test flask app
+      - name: Test flask web app
         run: |
-          pytest --random-order asreview/webapp/tests
+          pytest --random-order --ignore=asreview/webapp/tests/integration_tests asreview/webapp/tests
```

```suggestion
          pytest --random-order asreview/webapp/tests
```

This should work now with the pytest.ini file.